lib/experimental/get-global-styles-and-settings.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * API to interact with global settings & styles.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Repeated logic from `get_default_block_editor_settings` function. When implemented into core,
+ * remove logic from `get_default_block_editor_settings` and simple call this function instead.
+ *
+ * @return array
+ */
+function gutenberg_get_legacy_theme_supports_for_theme_json() {
```

Comment:
Oh, sorry for the confusion. This code should live in `lib/compat/wordpress-6.2` or `lib/compat/wordpress-6.1`, depending on the version we aim to backport it to. I only mentioned the resolver in my comment about the experimental folder.